ts/sdk/src/competitionClient.ts

```diff
@@ -413,9 +417,43 @@ export class CompetitionsClient {
 
 	public getCompetitionPublicKey(name: string): PublicKey {
 		const encodedName = encodeName(name);
-		return getCompetitionAddressSync(
-			this.program.programId,
-			encodedName
+		return getCompetitionAddressSync(this.program.programId, encodedName);
+	}
+
+	public async getCompetitionDetails(
+		competition: PublicKey,
+		insuranceFundVaultBalance: BN
+	) {
+		const competitionAccount = await this.program.account.competition.fetch(
+			competition
 		);
+
+		const quoteSpotMarketAccount = this.driftClient.getQuoteSpotMarketAccount();
+		const protocolOwnedShares =
+			quoteSpotMarketAccount.insuranceFund.totalShares.sub(
+				quoteSpotMarketAccount.insuranceFund.userShares
+			);
+
+		const protocolOwnedBalance = unstakeSharesToAmount(
+			protocolOwnedShares,
+			quoteSpotMarketAccount.insuranceFund.totalShares,
+			insuranceFundVaultBalance
+		);
+
+		const maxPrize = protocolOwnedBalance
```

Comment:
I'm a little confused by the calculation of the max prize here. this is the result I received from this API, compared to `arbJEWqPDYfgTFf3CdACQpZrk56tB6z7hPFc6K9KLUi`'s IF vault

<img width="294" alt="Screenshot 2023-10-09 at 11 51 00 AM" src="https://github.com/drift-labs/drift-competitions/assets/12388321/e035d96f-039a-463d-aa7c-5fb19dcbc6a0">

<img width="622" alt="Screenshot 2023-10-09 at 11 50 52 AM" src="https://github.com/drift-labs/drift-competitions/assets/12388321/83114ba2-c289-40d3-810e-a30b5a10c52a">

Reply:
i think he mighta forgot to divid by the maxSponsorFraction precision

Reply:
pushed PERCENTAGE_PRECISION div